mlflow/pytorch/__init__.py

```diff
@@ -128,6 +128,13 @@ def log_model(
     """
     Log a PyTorch model as an MLflow artifact for the current run.
 
+        .. warning::
+
+            Log the model with signature to avoid inference errors. Pytorch float precision default
+            is float32, while numpy float precision default is float64. Adding the signature will
+            ensure correct precision is used when the model is logged and the model can be queried
+            accurately.
+
```

Comment:
WDYT of:
Adding the signature will ensure correct precision is used when the model is logged and the model can be queried accurately.
--> Include the signature to ensure that the model is logged with the correct precision and produces accurate results.